java/arcs/core/data/expression/Stringifier.kt

```diff
@@ -84,11 +86,13 @@ class ExpressionStringifier(val parameterScope: Expression.Scope = ParameterScop
     expr.qualifier.accept(this, ctx) + "\norderby " + expr.selectors.joinToString(
       separator = ", "
     ) { sel ->
-      sel.expr.accept(this, ctx) + (if (sel.descending) {
-        " descending"
-      } else {
-        ""
-      })
+      sel.expr.accept(this, ctx) + (
+        if (sel.descending) {
+          " descending"
+        } else {
+          ""
+        }
+        )
```

Comment:
this seems to be a running bug, it doesn't know how to format multi-line subexpressions in a parenthesis